lib/elastic_whenever/task/rule.rb

```diff
@@ -8,25 +8,34 @@ class Rule
 
       class UnsupportedOptionException < StandardError; end
 
-      def self.fetch(option)
-        client = Aws::CloudWatchEvents::Client.new(option.aws_config)
-        client.list_rules(name_prefix: option.identifier).rules.map do |rule|
-          self.new(
+      def self.fetch(option, rules: [], next_token: nil)
+        client = option.cloudwatch_events_client
+        prefix = option.identifier
+
+        Logger.instance.message("Fetching Rules for prefix: #{prefix} next_token: #{next_token}")
```

Comment:
This log output may be a bit verbose (I think it's useful for testing pagination).